examples/network/network_api_demo.py

```diff
@@ -33,8 +33,8 @@
 
 _VERBOSITY = 0  # how chatty the demo should be
 _SEED = 1956  # the random seed used throughout
-_DATA_PATH = "extra/hotgym/rec-center-hourly.csv"
-_OUTPUT_PATH = "test_output.csv"
+_DATA_PATH = "../prediction/data/extra/hotgym/rec-center-hourly.csv"
```

Comment:
You should not have to do this. Please undo this change. If it doesn't work, you may need to set `NTA_DATA_PATH` but it should default correctly.